common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -340,6 +340,15 @@ trait FeatureSwitches {
     exposeClientSide = false
   )
 
+  val AUSElectionSwitch = Switch(
```

Comment:
Would you be able to add a comment with a owner, like line 352 `Owner: Dotcom loyalty`